Sources/SwiftSyntaxBuilderGeneration/SyntaxBuildableChild.swift

```diff
@@ -31,17 +34,37 @@ extension Child {
   /// Generate a Swift expression that creates a proper SwiftSyntax node of type
   /// `type.syntax` from a variable named `varName` of type `type.buildable` that
   /// represents this child node.
-  func generateExprBuildSyntaxNode(varName: String, formatName: String) -> String {
+  func generateExprBuildSyntaxNode(varName: String, formatName: String) -> ExpressibleAsExprBuildable {
```

Comment:
I consider `ExpressibleAsExprBuildable` to be mostly used as an input type to functions, not really something that’s produced. Unless there’s a reason against it, I would prefer to return `ExprBuildable` here.

Reply:
Should we do this for all output types, i.e. also all of the properties? (I have no strong preference either way)